front/src/applications/operationalStudies/views/Scenario.tsx

```diff
@@ -59,36 +59,51 @@ export default function Scenario() {
   const selectedProjection = useSelector(getSelectedProjection);
   const allowancesSettings = useSelector(getAllowancesSettings);
 
-  const { data: project } = osrdEditoastApi.endpoints.getProjectsByProjectId.useQuery(
+  const {
+    data: project,
+    isError: isProjectError,
+    error: errorProject,
+  } = osrdEditoastApi.endpoints.getProjectsByProjectId.useQuery(
     {
       projectId: +projectId,
     },
-    { skip: !projectId || Number.isNaN(+projectId) }
+    { skip: !projectId }
+  );
+  const {
+    data: study,
+    isError: isStudyError,
+    error: errorStudy,
+  } = osrdEditoastApi.endpoints.getProjectsByProjectIdStudiesAndStudyId.useQuery(
+    { projectId: +projectId, studyId: +studyId },
+    {
+      skip: !projectId || !studyId || Number.isNaN(+studyId),
```

Comment:
I don't really like the fact that we test whether studyId is a number or not and that we convert it to number several times. It would be better to do it once, and then to manipulate a variable which can be either a number or undefined.
Here is a suggestion of code for studyId:
```
 const { studyId: urlStudyId } = useParams() as SimulationParams;
const studyId = useMemo(
    () => (urlStudyId && !Number.isNaN(+urlStudyId) ? Number(urlStudyId) : undefined),
    [urlStudyId]
  );
```
Then you can manipulate studyId without testing everywhere if it's a number + converting it (only check whether it is undefined or not). It's the same for the other variables (projectId and scenarioId) and files (Project, Scenario and Study)

```suggestion
      skip: !projectId || !studyId
```

Reply:
If it's undefined it means that the parameters is not define in the url so the router will not handle the route and make a redirection.